postman/schemas/paths/token.yaml

```diff
@@ -0,0 +1,41 @@
+# /token
+get:
+    summary: Get all token if current user is admin.
+    operationId: getAllToken
+    responses:
+        "200":
+            description: Successfull operation
+            content:
+                application/json:
+                    schema:
+                        type: array
+                        items:
+                            type: string
+                            format: uuid
+        "401":
+            description: API token missing
+        "403":
+            description: No rights to see the requested content
+put:
+    summary: Create new token with specified permission specified in body if current user is admin
+    operationId: createToken
+    requestBody:
+        required: true
+        content:
+            application/json:
+                schema:
+                    type: object
+                    properties:
+                        permission:
+                            type: string
+    responses:
+        "201":
+            description: Successfully created new token
+            content:
+                application/json:
+                    schema:
+                        $ref: "../schemas/Token.yaml#/components/schema/Token"
```

Comment:
The token itself ~~must~~ should be generated on the server to make sure that the token has a high-enough entropy to be secure

Reply:
Thanks for the reminder 👍 I'll keep that in mind and I hope I can request a review from you when I implement this section 😇